catalogs/README.md

```diff
@@ -31,10 +31,10 @@ If your project requires additional python dependencies, add them to the `requir
 | HAND                              | s3://hand-data/catalog.tar.gz                                                                      | Height Above Nearest Drainage (derived from https://cfim.ornl.gov/data/)             |
 | Mississippi monthly water         | s3://global-surface-water/monthly/mississippi-river/global-water-monthly-catalog.tar.gz            | JRC Global Surface Water monthly observations of the Mississippi river region        |
 | Sen1Floods11                      | s3://sen1floods11-data/stac-catalog.zip                                                            | Sen1Floods11 dataset (https://github.com/cloudtostreet/Sen1Floods11)                 |
-| Sen1Floods11 s1weak ML            | s3://sen1floods11-data/mldata_s1weak_sen1.tar.gz                                                   | Sen1Floods11 imagery + labels (weakly trained s1 labels)                             |
-| Sen1Floods11 s2weak ML            | s3://sen1floods11-data/mldata_s2weak_sen1.tar.gz                                                   | Sen1Floods11 imagery + labels (weakly trained s2 labels)                             |
-| Sen1Floods11 training w/ HAND     | s3://sen1floods11-data/mldata_hand_sen1.tar.gz                                                     | Sen1Floods11 imagery with HAND + Sen1Floods11 labels                                 |
+| Sen1Floods11 s1weak ML            | s3://sen1floods11-data/mldata_s1weak_sen1.tar.gz                                                   | Sen1Floods11 imagery (w/ weak labels derived from Sentinel-1)                        |
```

Comment:
@moradology I changed the descriptions of this and the next two STACs in this table, and the location of the last STAC in this table.